packages/api/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "backend for LiteFarm",
   "main": "./api/src/server.js",
   "scripts": {
-    "test": "jest --runInBand",
```

Comment:
@MwayaB Changes to package.json often have a wide impact. Best to make them, when necessary, in consultation (or at least notification to) others.
In this particular case, I don't think you need or want to remove that. I was running only the modified test file with the command line `npx jest --runInBand ./tests/task.test.js`
That flag prevents jest from running tests in parallel; I don't know if the tests will work correctly if they are executed in parallel.